packages/simple-hub/start-dev-server.ts

```diff
@@ -13,18 +14,26 @@ async function setupGanacheAndContracts() {
 }
 
 async function startLocalFirebaseServer() {
+  const firebaseLog = log.child({context: 'firebase'});
+  firebaseLog.info('Starting server');
   const server = new FirebaseServer(5555, 'localhost');
+  firebaseLog.info('Started');
 
-  const closeServer = async () => await server.close();
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  const closeServer = (msg: string) => async (err?: any) => {
+    firebaseLog.info({err}, 'Closing server: %s', msg);
```

Comment:
```suggestion
    firebaseLog.info({err}, `Closing server: ${msg}`);
```

Reply:
> It would be nice if there was an easy way to interlace hub logs with browser (wallet and w3t) logs.

You mean `cat`? 😛 
```
cat simple-hub.log browser.log | jq -s -c 'sort_by(.time) | .[]' > all.log
```